src/components/ContactsPage.js

```diff
@@ -30,6 +31,22 @@ class ContactsPage extends React.Component {
     }
   }
 
+  areContactsWithLikes = () => {
+    const { contacts } = this.state;
+    return contacts.some(item => item.likes);
+  };
+
+  clearAllContactsLikes = () => {
+    const { contacts } = this.state;
+    contacts.forEach((item) => {
```

Comment:
You are mutating the contacts array here. Go for an immutable solution:
`const updatedContacts = contacts.map(contact => ({...contact, likes: 0}));`